components/layout_2020/dom_traversal.rs

```diff
@@ -409,12 +409,20 @@ where
                         vec.push(PseudoElementContentItem::Text(s.to_string()));
                     },
                     ContentItem::Attr(attr) => {
+                        use selectors::Element;
```

Comment:
Please put this import at the top of the file. If you need to rename it due to a naming conflict, it could be called `SelectorsElement`.

Reply:
I have applied the recommendations, was following how does the `selectors::Element` was imported previously and assumed it was imported like these to keep it in the scope. 

What is the convention of naming import like these if there are no conflicts? Should one kept it as `SelectorsElement` which will differentiate it with the other more well known `Element`?